src/svc-managed-functions.yml

```
@@ -1,9 +1,9 @@
 openapi: '3.0.0'
 info:
-  title: Influxdata Managed Functions CRUD API
+  title: InfluxData Managed Functions CRUD API
   version: 0.1.0
 servers:
-  - url: '/'
+  - url: '/api/v2'
```

This is incorrect, but raises an issue with the readme. anything in `/contracts/svc` defines an "internal" api and will likely have a different value for the `servers` key. This needs to be reverted.
```suggestion
  - url: '/'
```